daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/Functions.tdml

```diff
@@ -11808,7 +11808,7 @@
     </tdml:document>
     <tdml:infoset>
       <tdml:dfdlInfoset>
-        <ex:dfdlHexBinary02>D0</ex:dfdlHexBinary02>
+        <ex:dfdlHexBinary02>0208</ex:dfdlHexBinary02>
```

Comment:
Why is this change occurring?

Reply:
The original test assumed the behavior that a string of `"208"` gets converted to an integer of `208` and then converted to `0xd0` but the `208` should be parsed as hex digits and result in `0x0208`. A leading zero was added as the hexBinary constructor enforces an even number of digits.